packages/components/select/src/option.vue

```diff
@@ -51,6 +46,16 @@ export default defineComponent({
 
   setup(props) {
     const ns = useNamespace('select')
+
+    const containerKls = computed(() => [
+      ns.be('dropdown', 'item'),
+      ns.is('disabled', unref(isDisabled)),
+      {
+        ['selected']: unref(itemSelected),
+        ['hover']: unref(hover),
```

Comment:
There seems to be no need for square brackets here.

Reply:
updated